eng/build.py

```diff
@@ -1,207 +1,211 @@
-# SPDX-License-Identifier: Apache-2.0

```

Comment:
Don't know why this diff is so terrible.  Just changed all the pytest executions to specify the "tests" directory.